packages/firestore/CHANGELOG.md

```diff
@@ -1,50 +1,74 @@
 # Unreleased
-- [fixed] Fixed an issue that may have prevented the client from connecting 
+
+## 1.16.0
+
+### Minor Changes
+
+- [`39ca8ecf`](https://github.com/firebase/firebase-js-sdk/commit/39ca8ecf940472159d0bc58212f34a70146da60c) [#3254](https://github.com/firebase/firebase-js-sdk/pull/3254) Thanks [@thebrianchen](https://github.com/thebrianchen)! - Added support for `set()` with merge options when using `FirestoreDataConverter`.
+
+* [`877c060c`](https://github.com/firebase/firebase-js-sdk/commit/877c060c47bb29a8efbd2b96d35d3334fd9d9a98) [#3251](https://github.com/firebase/firebase-js-sdk/pull/3251) Thanks [@schmidt-sebastian](https://github.com/schmidt-sebastian)! - Re-adding the ReactNative bundle, which allows Firestore to be used without `btoa`/`atob` Polyfills.
+
+### Patch Changes
+
+- [`a754645e`](https://github.com/firebase/firebase-js-sdk/commit/a754645ec2be1b8c205f25f510196eee298b0d6e) [#3297](https://github.com/firebase/firebase-js-sdk/pull/3297) Thanks [@renovate](https://github.com/apps/renovate)! - Update dependency typescript to v3.9.5
```

Comment:
@Feiyang1 Is it possible to remove the "thanks" part? It is nice for external contributors, but not sure if we need it for team members.

We also might not want to add release notes for updates to dev dependencies (and maybe also all `@firebase` dependencies).

Reply:
Yes. We will need to write a custom generator. I'm not sure how to check for team member though, so we will probably completely remove the thanks part. I personally don't mind the thanks part, but if you feel strongly about it, I can make a custom generator.

> We also might not want to add release notes for updates to dev dependencies

Generally speaking, devDependencies should not be included. Typescript and some build tools(e.g. rollup) are special cases because they could affect the content of the build artifacts, so I think their upgrades should be included in the changelog.

> maybe also all @firebase dependencies

They are `dependencies`, so their upgrades should be included in the changelog. There could be cases where only `dependencies` are updated with no other changes. It should trigger a release in firestore and we want to show it in the changelog.

Reply:
The reason I am kind of hesitant to include updates on internal dependencies is that it feels like clutter to me (but that is a personal opinion). 
Android and iOS do not include them and we also exclude dependency updates for `@google-cloud/firestore`.